cdflow_commands/plugins/aws_lambda.py

```diff
@@ -174,11 +174,13 @@ def create(self):
 
     def _zip_up_component(self):
         logger.info('Zipping up ./{} folder'.format(self._component_name))
+        top_level = os.getcwd()
         with ZipFile(self._component_name + '.zip', 'w') as zipped_folder:
             for dirname, subdirs, files in os.walk(self._component_name):
-                zipped_folder.write(dirname)
+                os.chdir(dirname)
```

Comment:
Be careful with using chdir, sometimes it gets tricky. If there is an exception writing the file, your program would have changed directory. You can use context_manager https://pythonadventures.wordpress.com/2013/12/15/chdir-a-context-manager-for-switching-working-directories/

But as this is a one off execution and any exception would abort the program, probably it is OK.